bindings/matrix-sdk-crypto-js/Cargo.toml

```diff
@@ -45,9 +46,11 @@ matrix-sdk-indexeddb = { version = "0.2.0", path = "../../crates/matrix-sdk-inde
 matrix-sdk-qrcode = { version = "0.4.0", path = "../../crates/matrix-sdk-qrcode", optional = true }
 ruma = { workspace = true, features = ["js", "rand", "unstable-msc2677"] }
 serde_json = { workspace = true }
-tracing = { workspace = true, optional = true }
+tracing = { workspace = true }
```

Comment:
because https://github.com/matrix-org/matrix-rust-sdk/pull/1704/files#diff-eaaa2673496a447cd817cdf32dc49c9d2a40470f0fa9951f6c466eb6cfb5d3cfR813 is in non-optional code.

Presumably I should remove the dependency from the optional feature?

Reply:
Yeah, the feature shouldn't depend on `tracing` anymore.